resources/helm/development.yaml

```diff
@@ -1670,7 +1673,10 @@ mod-ldp:
     maxUnavailable: 1
 mod-reporting:
   extraEnvVars: []
-  integrations: {}
+  integrations:
+    okapi:
```

Comment:
mod-reporting, don't need this integration

Reply:
Done